packages/admin-ui/apollo/server/resolvers.ts

```diff
@@ -199,3 +197,28 @@ export const resolvers = {
     },
   },
 };
+
+const projectResolver = new ProjectResolver();
+const modelResolver = new ModelResolver();
+
+const resolvers = {
+  JSON: GraphQLJSON,
+  Query: {
+    listDataSourceTables: projectResolver.listDataSourceTables,
+    autoGenerateRelation: projectResolver.autoGenerateRelation,
+    listModels: modelResolver.listModels,
+  },
```

Comment:
May I ask the `usableDataSource` API will provide by backend or not?
Bcos we didn't remark in the web API spec, I think it should add in?

Reply:
after discussion, we will remove the usableDataSource and FE will take care of it